paymentrails/paymentrails.csproj

```diff
@@ -26,11 +26,11 @@
     <RootNamespace>PaymentRails</RootNamespace>
     <PackageId>PaymentRails.NetCore.SDK</PackageId>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <Version>1.0.9</Version>
-    <PackageDescription>The official Payment Rails NET Core and NET Standard SDK to interface with the Payment Rails API (https://www.paymentrails.com/)</PackageDescription>
+    <Version>1.0.10</Version>
+    <PackageDescription>The official Trolley NET Core and NET Standard SDK to interface with the Trolley API (https://www.trolley.com/)</PackageDescription>
     <Copyright>Copyright 2020</Copyright>
-    <Authors>Jesse Tremblay, Jesse Silber, Joshua Cunningham</Authors>
-    <PackageTags>netcore sdk PaymentRails</PackageTags>
+    <Authors>Jesse Tremblay, Jesse Silber, Joshua Cunningham, Aman Aalam</Authors>
+    <PackageTags>netcore sdk PaymentRails Trolley</PackageTags>

```

Comment:
Want to keep PaymentRails here?

Reply:
It will be going away shortly, in our next update.
We'll publish on a new path on Nuget as well